desktop/app/paths.js

```diff
@@ -0,0 +1,39 @@
+import {app} from 'electron'
+import path from 'path'
+import os from 'os'
+
+function appPath () {
+  // For testing when running manually via npm start
+  // return '/Applications/Keybase.app/Contents/Resources/app/'
+  // For testing running from DMG
+  // return '/Volumes/Keybase/Keybase.app/Contents/Resources/app/'
+  return app.getAppPath()
+}
+
+// Path to bundle directory, e.g. /Applications/Keybase.app (darwin only)
+export function appBundlePath () {
+  if (os.platform() !== 'darwin') return ''
```

Comment:
Go taught me to prefer empty string, but it is a little weird to do that in javascript. Changed to nulls and doing truthy checks.